kyaml/fix/fixsetters/lookupupgradekio.go

```diff
@@ -74,6 +74,11 @@ func (l *UpgradeV1Setters) Filter(input []*yaml.RNode) ([]*yaml.RNode, error) {
 	for _, subst := range substitutions {
 		l.Substitutions = append(l.Substitutions, *subst)
 	}
+
+	sort.Slice(l.Substitutions, func(i, j int) bool {
```

Comment:
Why do we need this?

Reply:
For the output in openApi to be deterministic. As we are using the set, the substitutions are randomized in order. We need this for tests to pass consistently.